draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5902,11 +5902,13 @@ Expect: 100-continue
 </t>
 <t>
    An origin server that receives an If-Unmodified-Since header field &MUST;
-   evaluate the condition prior to performing the method
-   (<xref target="evaluation"/>).
-   The origin server &MUST-NOT; perform the requested method if the selected
-   representation's last modification date is more recent than the date
-   provided in the field value; instead the origin server &MUST; respond with either
+   evaluate the condition (<xref target="evaluation"/>) prior to performing
+   an unsafe method (<xref target="safe.methods"/>).
+</t>
```

Comment:
This changes the requirement to apply only to unsafe methods. Is this really intended?

Reply:
Yes. Requiring servers to support If-Unmodified-Since on all resources for all request methods isn't viable. Happy to take it to the list if you're concerned, though.

Reply:
Looking at this again, I don't think this change is supportable; it works that way in a couple of major servers that I spot checked. So, change backed out.